docs/charts_tips_and_tricks.md

```diff
@@ -0,0 +1,71 @@
+# Chart Development Tips and Tricks
+
+This guide covers some of the tips and tricks Helm chart developers have
+learned while building production-quality charts.
+
+## Quote Strings, Don't Quote Integers
+
+When you are working with string data, you are always safer quoting the
+strings than leaving them as bare words:
+
+```
+name: {{.Values.MyName | quote }}
+```
+
+But when working with integers _do not quote the values._ That can, in
+many cases, cause parsing errors inside of Kubernetes.
+
+```
+port: {{ .Values.Port }}
+```
+
+## Using the 'include' Function
```

Comment:
Ah I see you've documented it here. Perhaps we can link to it from the section in charts.md